xbmc/addons/AddonSystemSettings.h

```diff
@@ -21,6 +21,9 @@ const int AUTO_UPDATES_ON = 0;
 const int AUTO_UPDATES_NOTIFY = 1;
 const int AUTO_UPDATES_NEVER = 2;
 
+const int UPDATEMODE_PREFER_TRUSTED = 0;
+const int UPDATEMODE_ANY_REPOSITORY = 1;
```

Comment:
This should be an enum class which sets the values. E.g.:

```cpp
  enum class AddonTrustedUpdateMode : int
  {
    PREFER_TRUSTED = 0,
    ANY_REPOSITORY = 1
  };
```

Defining const ints would be the old way of doing this. Using an enum mean it can be type safe.